pkg/registry/discovery/endpointslice/strategy.go

```diff
@@ -222,3 +231,22 @@ func warnOnDeprecatedAddressType(addressType discovery.AddressType) []string {
 	}
 	return nil
 }
+
+// warnOnBadIPs returns warnings for bad IP address formats
+func warnOnBadIPs(eps *discovery.EndpointSlice) []string {
+	// Don't bother checking for bad IPs if it looks like the request is coming from
+	// one of our controllers.
+	if eps.Labels[discoveryv1.LabelManagedBy] == "endpointslice-controller.k8s.io" ||
+		eps.Labels[discoveryv1.LabelManagedBy] == "endpointslicemirroring-controller.k8s.io" {
```

Comment:
wonder if we should export this constants and references directly 

```
grep -r endpointslice-controller.k8s pkg/
pkg/controller/endpointslice/endpointslice_controller.go:       controllerName = "endpointslice-controller.k8s.io"
aojea@aojea:~/src/kubernetes$ grep -r endpointslicemirroring-controller.k8s pkg/
pkg/controller/endpointslicemirroring/endpointslicemirroring_controller.go:     controllerName = "endpointslicemirroring-controller.k8s.io"
```

Reply:
Yeah, I could do that...